esp/services/ws_packageprocess/ws_packageprocessService.cpp

```diff
@@ -287,13 +298,17 @@ class PackageMapUpdater
         process.set(name);
         buildPkgSetId(pkgSetId, process);
     }
-    void setUser(const char *user, const char *password)
+    void setUser(const char *user, const char *password, IEspContext *context)
     {
-        if (user && *user && *password && *password)
+        if (user && *user)
```

Comment:
The original test didn't check for password == null, but now we're not testing for null nor blank, just making sure that's by design.